gamemode/modules/animations/sh_animations.lua

```diff
@@ -29,7 +29,7 @@ end
 local function physGunCheck(ply)
     local hookName = "darkrp_anim_physgun_"..ply:EntIndex()
     hook.Add("Think", hookName, function()
-        if IsValid(ply) and ply:Alive() and ply:GetActiveWeapon():IsValid() and ply:GetActiveWeapon():GetClass() == "weapon_physgun" and ply:KeyDown(IN_ATTACK) then
+        if IsValid(ply) and ply:Alive() and ply:GetActiveWeapon():IsValid() and ply:GetActiveWeapon():GetClass() == "weapon_physgun" and ply:KeyDown(IN_ATTACK) and (not ply:GetAllowWeaponsInVehicle() and ply:InVehicle()) then
```

Comment:
This will break the animation when the player is not in a vehicle, but `ply:GetAllowWeaponsInVehicle()` return `false`.

The correct condition would be `and (ply:GetAllowWeaponsInVehicle() or not ply:InVehicle())`. That way, the player is allowed to do the animation when they can hold the physgun in a vehicle, or they are not in a vehicle.